### PR TITLE
Make sure message is a string.

### PR DESCRIPTION
**Pull request by vmihailenco**
Hopefully fixes https://github.com/airbrake/airbrake-js/issues/197

### CODE REVIEW

File: src/processors/stack.coffee [hunk 1]
```diff
@@ -179,7 +179,7 @@ processor = (e, cb) ->
     })
 
   if e.message?
-    msg = e.message
+    msg = String(e.message)
```
Comment (zefer) [inline src/processors/stack.coffee]: What do you suspect `e.message` is in #197? Or is it a stab in the dark? :)

Comment (vmihailenco) [inline src/processors/stack.coffee]: Because it says "e.message is an event in this case."

Comment (zefer) [inline src/processors/stack.coffee]: Can we test for that?

**Comment by vmihailenco** [comment]
Added a test. PTAL

**Comment by zefer** [comment]
:+1: LGTM

**Pull request merged**